### DISCUSSION TITLE
Queuing doesn't stop at Dify.

### DISCUSSION THREAD

**Discussion by bakva9** [langgenius/dify#10677] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I am building my Dify on Xserver (installed directly on the server, no Docker) and after updating to the latest version of Dify, I am having problems with Knowledge registrations stopping in queue and not being able to register. How can this be resolved?

**Comment by luoyutao03** [comment]
I have a similar issue to yours.I deployed DIFY using DOCKER DESKTOP and when uploading documents to the knowledge base, the status always shows is  queueing.I am currently unable to handle it。

**Comment by zhu25244** [comment]
I'm also having this problem, the build file keeps showing up in the queue, but I can't get to the parsing. Some people have been able to run it on my laptop, does anyone know what the problem is?

**Comment by misdag-tju** [comment]
I have the same problem. It has persisted for several days and remains unresolved. I'm unsure of the underlying issue。

**Comment by David030111** [comment]
I have a similar issue to yours.I deployed DIFY using DOCKER DESKTOP and when uploading documents to the knowledge base, the status always shows is queueing.I am currently unable to handle it。

**Comment by crazywoola** [comment]
https://github.com/langgenius/dify/issues/10555#issuecomment-2470651673

Maybe it's related to this issue, without more details, I can not do much about it.
